scripts/tests/calibnet_stateless_mode_check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script tests the stateless mode of a forest node
+
+source "$(dirname "$0")/harness.sh"
+
+forest_init_stateless
+
+echo "Verifying the heaviest tipset to be the genesis"
+HEAD_CID=$($FOREST_CLI_PATH chain head | jq -r '.[0]')
+assert_eq "$HEAD_CID" "bafy2bzacecyaggy24wol5ruvs6qm73gjibs2l2iyhcqmvi7r7a4ph7zx3yqd4"
+
+STATELESS_NODE_ADDRESS=$($FOREST_CLI_PATH net listen | tail -n 1)
+echo "Stateless node address: $STATELESS_NODE_ADDRESS"
+STATELESS_NODE_PEER_ID=$(echo "$STATELESS_NODE_ADDRESS" | cut --delimiter="/" --fields=7 --zero-terminated)
```

Comment:
It might be worth commenting on an expected format with an example multiaddress.

Reply:
Done